VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts

```diff
@@ -265,17 +280,15 @@ export class ParticipantsPanelComponent implements OnInit, OnDestroy {
             const pats = await this.videoWebService.getParticipantsByConferenceId(this.conferenceId);
```

Comment:
Can these be removed now and re[;replaced with the participant service?

Reply:
Do we need to do some additional work to replicate the sorting in the setParticipants method in this component?

Reply:
I'm not sure what is the best way to go about that to be honest. Is there other components that need different ways of sorting across the application or is there just one way they are sorted? A sorting service might be ideal maybe?